lib/github_changelog_generator/fetcher.rb

```diff
@@ -123,7 +124,11 @@ def fetch_closed_issues_and_pr
     def fetch_closed_pull_requests
       pull_requests = []
       begin
-        response = @github.pull_requests.list @options[:user], @options[:project], state: "closed"
+        if @options[:release_branch].nil?
+          response = @github.pull_requests.list @options[:user], @options[:project], state: "closed"
+        else
+          response = @github.pull_requests.list @options[:user], @options[:project], state: "closed", base: @options[:release_branch]
+        end
```

Comment:
It's like a [flat_map](http://apidock.com/ruby/Enumerable/flat_map) with some paging-handling and logging output is all this method is.